<launch>

  <arg name="launch_prefix" default="" /> <!-- set to value="gdbserver localhost:10000" for remote debugging -->
  <arg name="node_namespace" default="shitcam" />
  <arg name="camera_name" default="/record" />
  <arg name="camera_frame" default="record" />
  <arg name="image_topic" default="image_rect" />

  <!-- Start the camera node-->
  <node name="record" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video6" />
    <param name="image_width" value="800" />
    <param name="image_height" value="600" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="record" />
<!--    <param name="camera_info_url" value="file:////home/roblab20/osher3_workspace/src/tactile_insertion/config/camera1.yaml"/>-->
  </node>

  <!--Convert image_raw to undistorted cam-->
<!--  <node pkg="image_proc" type="image_proc" name="image_proc_node" ns="record" />-->

</launch>
